lib/phut/phost.rb

```diff
@@ -4,13 +4,16 @@ module Phut
   # An interface class to phost emulation utility program.
   class Phost
     attr_reader :ip
+    attr_reader :name
+    attr_accessor :interface
 
-    def initialize(ip_address)
+    def initialize(ip_address, name = nil)
       @ip = ip_address
+      @name = name || @ip
     end
 
     def run
-      system "sudo #{executable} #{options.join ' '}"
+      system("sudo #{executable} #{options.join ' '}")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.